src/jdk.jdwp.agent/share/native/libjdwp/util.c

```diff
@@ -1918,9 +1924,43 @@ map2jvmtiError(jdwpError error)
     return AGENT_ERROR_INTERNAL;
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static jvmtiExtensionEventInfo *
+find_ext_event(jvmtiEnv *jvmti, const char *ename)
+{
+    jint extCount = 0;
+    jvmtiExtensionEventInfo *extList = NULL;
+
+    jvmtiError err = JVMTI_FUNC_PTR(jvmti, GetExtensionEvents)
+                            (jvmti, &extCount, &extList);
+    if (JVMTI_ERROR_NONE == err) {
+        for (int i = 0; i < extCount; i++) {
+            if (NULL != strstr(extList[i].id, ename)) {
+                return &extList[i];
+            }
+        }
+    } else {
+        ERROR_MESSAGE(("Error in JVMTI GetExtensionEvents: %s(%d)\n", jvmtiErrorText(err), err));
```

Comment:
GetExtensionEvents -> GetExtensionEvent or find_ext_event

Reply:
`GetExtensionEvents` is the correct JVMTI function name.